plugins/inputs/consul/consul.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/consul/api"
 
 	"github.com/influxdata/telegraf"
+	telegrafConfig "github.com/influxdata/telegraf/config"
```

Comment:
Why do you need an alias? If you really need it the new convention is snake-case i.e.
```suggestion
	telegraf_config "github.com/influxdata/telegraf/config"
```

Reply:
There is a conflict in `createAPIClient` method otherwise.

Reply:
Then please use snake case.